### DISCUSSION TITLE
FleetDm queries

### DISCUSSION THREAD

**Discussion by Azriel-l** [Security-Onion-Solutions/securityonion#10295] [Unsupported Versions]
Hi guys ..
i've  installed securtiy onion 2.3.240
and i have some issues with osquery and fleetDm

1. downloaded and installed the agents from securityonion 
to my windows 2019 server
but some queries give me this error: 
osquery_version :5.8.2 error: Error details require osquery 4.4.0+ (Launcher does not provide error details) 

any ideas?

2. software quering is disabled how do i enable it?

**Comment by dougburks** [comment]
Last week, we released Security Onion 2.3.250 which includes FleetDM 4.31.1:
https://blog.securityonion.net/2023/05/security-onion-23250-now-available.html

Please try upgrading and see if it helps.
